pycipapi/cipapi_client.py

```diff
@@ -152,6 +162,18 @@ def submit_interpretation_flags(self, payload, case_id, case_version, **params):
         for flag in flags:
             yield InterpretationFlag(**flag)
 
+    def post_participant_interpreted_genome_raw(self, payload, participant_id, **params):
+        url = self.build_url(self.url_base, self.PARTICIPANTS_ENDPOINT, participant_id, 'interpreted-genome') + '/'
+        payload_json =  {
+                          "interpreted_genome_data": payload,
+                          "interpretation_service_name": "genomics_england_additional_findings"
```

Comment:
would it be better to have this as a variable with a default value?

Reply:
yes good idea